core/vmpu/src/kinetis/vmpu_kinetis_mem.c

```diff
@@ -63,20 +63,30 @@ void vmpu_mem_switch(uint8_t src_box, uint8_t dst_box)
 
 void vmpu_mem_init(void)
 {
+    /* UART DMA is bus master 2 (configured through M2UM). ENET DMA is bus
+     * master 3 (configured through M3UM). */
+    static const uint32_t bmum_read_bit = (1 << 2);
+    static const uint32_t bmum_write_bit = (1 << 1);
+#define M3UM 18
```

Comment:
Revised to use only static consts named with the register name, concatenated with the bitfield name, concatenated with the bit name from the datasheet.